qdb/utils.py

```diff
@@ -186,7 +198,11 @@ def __call__(self, seconds, exception=None, green=False):
 
     ast.Global,
     ast.Pass,
-}
+
+    # Python 2 only
+    ast.Print if PY2 else None,
```

Comment:
idk, I could imagine a function that accepted either an ast node or None and tried to check if it was in this set, assuming None would never be there.  Far-fetched, but possible.  I would also just be confused if I was in a debugger and printed this set and saw None.